README.md

```diff
@@ -1,15 +1,245 @@
+* `--gadi-account` the Gadi project account you would like to bill service units to
+
+
+### NCI Gadi HPC
+
+Before running the pipeline you will need to load Nextflow and Singularity, both of which are globally installed modules on Gadi. You can do this by running the commands below:
+
+```
+module purge
+module load nextflow singularity
+```
+
+To execute this workflow on NCI Gadi HPC, you will need to specify the following flags to the default run command:
+
+```
+nextflow run main.nf --input sample.tsv --ref /path/to/ref --gadi-account <account> --whoami <username> -profile gadi
+```
+
+Please be aware that as of October 2023, NCI Gadi HPC queues do not have external network access. This means you will not be able to pull the workflow code base or containers if you submit your nextflow run command as a job on any of the standard job queues. NCI currently recommends you run your Nextflow head job either in a GNU screen or tmux session from the login node or submit it as a job to the copyq.
+
+The NCI Gadi config currently runs all tasks apart from the rehead processes on the normal queue. This config uses the `--gadi-account` flag to assign a project code to all task job submissions for billing purposes. The version of Nextflow installed on Gadi has been modified to make it easier to specify resource options for jobs submitted to the cluster. See NCI's [Gadi user guide](https://opus.nci.org.au/display/DAE/Nextflow) for more details.
+
+
+Please use the PBS script `PBS_gadi_runPipeline.sh` present in the main `Somatic-shortV-nf/` directory for testing the pipeline with test datasets. 
+**NOTE**: The pipeline is currently being tested without providing the   in the main command.
+
+
+## Benchmarking
+
+Coming soon!
+
+
+## Workflow summaries
+### Metadata
+
+|metadata field     | GermlineStructuralV-nf / v1.0     |
+|-------------------|:--------------------------------- |
+|Version            | 1.0                               |
+|Maturity           | under development                 |
+|Creators           | Tracy Chew, Cali Willet,Nandan Deshpande                    |
+|Source             | NA                                |
+|License            | GNU General Public License v3.0   |
+|Workflow manager   | NextFlow                          |
+|Container          | See component tools               |
+|Install method     | NA                                |
+|GitHub             | https://https://github.com/Sydney-Informatics-Hub/Somatic-shortV-nf                            |
+|bio.tools          | NA                                |
+|BioContainers      | NA                                |
+|bioconda           | NA                                | 
+
+### Component tools
+
+To run this pipeline you must have Nextflow and Singularity installed on your machine. All other tools are run using containers. 
+
+|Tool         | Version  |
+|-------------|:---------|
+|Nextflow     |>=20.07.1 |
+|Singularity  |          |
+|SnpEff       |     |
```

Comment:
Missing GATK and versions of other tools used

Reply:
Added